crates/modelardb_server/src/storage/compressed_data_manager.rs

```diff
@@ -154,11 +140,11 @@ impl CompressedDataManager {
 
     /// Insert the `compressed_segments` into the in-memory compressed data buffer for the table
     /// with `table_name`. If `compressed_segments` is saved successfully, return [`Ok`], otherwise
-    /// return [`IOError`].
+    /// return [`crate::error::ModelarDbServerError`].
```

Comment:
Maybe "[\`ModelarDbServerError\`]\(crate::error::ModelarDbServerError\)" instead. If it is changed here, also change it elsewhere in the file.

Reply:
Fixed.